selectable/templatetags/selectable_tags.py

```diff
@@ -13,5 +13,5 @@ def include_jquery_libs(version='1.11.2', ui='1.11.3'):
 
 
 @register.inclusion_tag('selectable/jquery-css.html')
-def include_ui_theme(theme='base', version='1.11.3'):
+def include_ui_theme(theme='smoothness', version='1.11.3'):
```

Comment:
Because http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/themes/base/jquery-ui.css does not exists.